src/cryptography/hazmat/primitives/serialization/ssh.py

```diff
@@ -572,13 +576,58 @@ def encode_private(
         f_priv.put_sshstr(f_keypair)
 
 
+class _SSHFormatSK:
+    def load_application(self, data) -> tuple[memoryview, memoryview]:
+        application, data = _get_sshstr(data)
+        return application, data
+
+
+class _SSHFormatSKEd25519(_SSHFormatEd25519, _SSHFormatSK):
```

Comment:
We pretty strictly avoid subclassing our code. A better approach would be to have a `load_application` function (no class), and then in `_SSHFormatSKEd25519.load_pulic` you can do `_SSHFormatEd25519().load_public()` followed by a call to it

Reply:
done - assuming subclassing _SSHFormatEd25519 is okay.